src/js/storage/storage.js

```diff
@@ -21,9 +21,10 @@ export class Store {
 
   set state(state) {
     if (state) {
-      this.storage.setItem(this.key, JSON.stringify(state));
-    }
-  }
+      //Do not stringify value if value is already is a string ; to avoid adding "" around the string value
+      this.storage.setItem(this.key, typeof state === 'string' ? state : JSON.stringify(state));
```

Comment:
This unfortunately breaks reading the values since plain strings are not parseable by `JSON.parse`. I believe we either need to store everything as JSON or nothing as JSON.

I need to parse the user token, which is stored as a plain string. I'll revert this change in my next PR.

Reply:
Ups, Good catch!